packages/app/src/app/pages/Dashboard/utils/getChildCollections.js

```diff
@@ -0,0 +1,15 @@
+import getDirectChildren from './get-direct-children';
+
+export default (folders, path = '/') => {
```

Comment:
Could this become a named function? That way we have nicer auto-completions (eg. `getChildCollections` would autocomplete and auto import).